tests/unit/converters/test_pandapower_converter_input.py

```diff
@@ -814,45 +834,47 @@ def test_create_pgm_input_transformers(mock_init_array: MagicMock, two_pp_objs,
-    converter._get_pp_attr.assert_any_call("trafo", "tap_max", 0)
-    converter._get_pp_attr.assert_any_call("trafo", "tap_pos", np.nan)
-    converter._get_pp_attr.assert_any_call("trafo", "parallel", 1)
-    converter._get_pp_attr.assert_any_call("trafo", "in_service", True)
-    converter._get_pp_attr.assert_any_call("trafo", "vk0_percent", np.nan)
-    converter._get_pp_attr.assert_any_call("trafo", "vkr0_percent", np.nan)
-    converter._get_pp_attr.assert_any_call("trafo", "mag0_percent", np.nan)
-    converter._get_pp_attr.assert_any_call("trafo", "mag0_rx", np.nan)
-    converter._get_pp_attr.assert_any_call("trafo", "si0_hv_partial", np.nan)
-    # converter._get_pp_attr.assert_any_call("trafo", "df")  #TODO add df in output conversions
+    converter._get_pp_attr.assert_any_call("trafo", "hv_bus", expected_type="u4")
+    converter._get_pp_attr.assert_any_call("trafo", "lv_bus", expected_type="u4")
+    converter._get_pp_attr.assert_any_call("trafo", "sn_mva", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "vn_hv_kv", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "vn_lv_kv", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "vk_percent", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "vkr_percent", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "pfe_kw", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "i0_percent", expected_type="f8")
+    converter._get_pp_attr.assert_any_call("trafo", "shift_degree", expected_type="f8", default=0.0)
+    converter._get_pp_attr.assert_any_call("trafo", "tap_side", expected_type="O", default=None)
+    converter._get_pp_attr.assert_any_call("trafo", "tap_neutral", expected_type="f8", default=np.nan)
+    converter._get_pp_attr.assert_any_call("trafo", "tap_min", expected_type="i4", default=0)
+    converter._get_pp_attr.assert_any_call("trafo", "tap_max", expected_type="i4", default=0)
+    converter._get_pp_attr.assert_any_call("trafo", "tap_pos", expected_type="f8", default=np.nan)
+    converter._get_pp_attr.assert_any_call("trafo", "parallel", expected_type="u4", default=1)
+    converter._get_pp_attr.assert_any_call("trafo", "in_service", expected_type="bool", default=True)
+    converter._get_pp_attr.assert_any_call("trafo", "vk0_percent", expected_type="f8", default=np.nan)
+    converter._get_pp_attr.assert_any_call("trafo", "vkr0_percent", expected_type="f8", default=np.nan)
+    converter._get_pp_attr.assert_any_call("trafo", "mag0_percent", expected_type="f8", default=np.nan)
+    converter._get_pp_attr.assert_any_call("trafo", "mag0_rx", expected_type="f8", default=np.nan)
+    converter._get_pp_attr.assert_any_call("trafo", "si0_hv_partial", expected_type="f8", default=np.nan)
+    # converter._get_pp_attr.assert_any_call('trafo', 'df', expected_type='f8')  #TODO add df in output conversions
```

Comment:
this entire line should be either removed or fixed. @nitbharambe  can you answer this?

Reply:
Dont remember this one. Need to dive deeper

Reply:
we can also leave it be... if i do a global search for TODO's there are many others (too many if you ask me, so let's discuss today)